Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:latest
+MAINTAINER nelluri@deis.com
+RUN apt-get update && apt-get -y install make python lsb-release unzip wget vim git curl jq
+RUN wget https://storage.googleapis.com/kubernetes-release/release/v1.0.3/bin/linux/amd64/kubectl
+RUN chmod +x kubectl
+RUN mv kubectl /usr/local/bin/
+WORKDIR /home
+RUN wget --no-check-certificate https://dl.bintray.com/deis/helm/helm-0.5.0+1689ee4-linux-amd64.zip
```

Comment:
--2016-03-07 16:44:14--  https://dl.bintray.com/deis/helm/helm-0.5.0+1689ee4-linux-amd64.zip
Resolving dl.bintray.com (dl.bintray.com)... 75.126.118.188, 108.168.243.150
Connecting to dl.bintray.com (dl.bintray.com)|75.126.118.188|:443... connected.
HTTP request sent, awaiting response... 302 
Location: https://akamai.bintray.com/ca/caaaa5d316408b36a741f2513f6d3d3889bc1ff7d2332bf74bc365c61918810c?__gda__=exp=1457369768~hmac=16ede66ae9dc0daee3af3f7112427ba1571f7b1c18bb71ced90099ff0b1b5688&response-content-disposition=attachment%3Bfilename%3D%22helm-0.5.0%2B1689ee4-linux-amd64.zip%22&response-content-type=application%2Fx-www-form-urlencoded [following]
--2016-03-07 16:44:15--  https://akamai.bintray.com/ca/caaaa5d316408b36a741f2513f6d3d3889bc1ff7d2332bf74bc365c61918810c?__gda__=exp=1457369768~hmac=16ede66ae9dc0daee3af3f7112427ba1571f7b1c18bb71ced90099ff0b1b5688&response-content-disposition=attachment%3Bfilename%3D%22helm-0.5.0%2B1689ee4-linux-amd64.zip%22&response-content-type=application%2Fx-www-form-urlencoded
Resolving akamai.bintray.com (akamai.bintray.com)... 104.71.78.240
Connecting to akamai.bintray.com (akamai.bintray.com)|104.71.78.240|:443... connected.
ERROR: cannot verify akamai.bintray.com's certificate, issued by '/C=NL/L=Amsterdam/O=Verizon Enterprise Solutions/OU=Cybertrust/CN=Verizon Akamai SureServer CA G14-SHA1':
  Unable to locally verify the issuer's authority.